ast.go

```diff
@@ -5,6 +5,7 @@ import (
 	"go/ast"
 	"go/format"
 	"go/token"
+	"strconv"
```

Comment:
Please adapt the commit message so that the subject is just "Fixed size array implementation" and the description states "Closes #48". The word "implementation" is redundant, and GH is trigged by "#<issue number>", it links the issue to this commit, and "closes" triggers GH to close the issue automatically.